### DISCUSSION TITLE
Syringe gun hits player

### DISCUSSION THREAD

**Discussion by Thor3005** [FakeFishGames/Barotrauma#14196] [Bug reports]
Edit: caused by mod called "BetterPoisons". Still it seems that **maybe** the devs had an oversight where the syringes need to have some "Don't hit syringe firer" property attached to them manually, so modders could forget about it or not know about it. Might be worth looking into a bit.

~~It seems the player holds the syringe gun so close to their body, the syringe, being at the back of the gun, spawns there and registers a hit with the player.

Precisely in my example, I was using the basic syringe gun, aimed horizontally, loaded with a single syringe of sufforin.

I'd suggest attaching a timer to it, where a few moments after firing, it can't hit the person that fired it.~~

**Comment by Regalis11** [comment]
Seems to be an issue with a mod, and doesn't occur in the vanilla game, closing.
